app/src/styles/base.css

```
@@ -441,6 +473,30 @@ h1, h2, h3, h4 { font-family: var(--font-heading); margin: 0; }
+ *
+ * The delay is only on the question that follows an answer, and it is the
+ * app's own stagger cap rather than a new number. It buys the beat that makes
+ * this read as a reply — the answer lands first, then the next question comes
+ * back — instead of both turns appearing together, which is what a form does.
+ *
+ * `backwards`, not `both`, for the same reason .row-in takes it: a filling
+ * animation outranks the cascade for the element's whole life, and these
+ * bubbles are re-rendered on every subsequent answer.
+ *
+ * Safe to transform: a bubble is a solid pane with no glass inside it. */
+@keyframes bubbleIn {
```

Stylelint (17.14.0)

[error] Expected keyframe name "bubbleIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)